### ISSUE TITLE
Failure to compile in fedora (using gcc/gfortran 8)

### ISSUE THREAD

**Issue by srio** [oasys-kit/shadow3#35]
with gfortran 8 the compilation of shadow_variables.f90 (and others) fails with the following error:

Error: Component 'file_xxx' of BIND(C) type at (1) must have length one

This is a change of behaviour in the char C/Fortran interface. See: 

https://gcc.gnu.org/bugzilla/show_bug.cgi?id=84885

See also this discussion: 

https://sourceforge.net/p/wsjt/mailman/message/36262831/

**Comment by srio** [comment]
This has been solved some time ago by Tom Schoonjans. Thanks a lot Tom. 

The changes are not yet included in the master. In the meantime, you can use Tom's version doing: 
```
git clone https://github.com/PaNOSC-ViNYL/shadow3.git

cd shadow3

git checkout gfortran8-fixes
```

**Comment by srio** [comment]
This is fixed in shadow3 22.8.20
